src/core/mrc_file.cpp

```diff
@@ -233,12 +234,24 @@ void MRCFile::ReadSlicesFromDisk(int start_slice, int end_slice, float *output_a
 					output_array[output_counter] = float(hi_4bits);
 					output_counter ++;
 				}
+			}
+			else
+			{
+				if (my_header.PixelDataAreSigned())
+				{
+					for (long counter = 0; counter < records_to_read; counter++)
+					{
+						output_array[counter] = float(temp_signed_char_array[counter] + 128);
```

Comment:
I guess this is converting it to signed?   I don't mind either way - but why not leave it signed?

Reply:
I tried it without the +128, if that what you mean, and the conversion didn't come out right. I tested with both signed and unsigned files (coming from IMOD's clip) and this is how the conversion worked (so that the volume output by cisTEM in float had the value matching the input from IMOD, so 1 came out as 1.0 and 0 came out as 0.0, etc)